.github/workflows/build-lint-test.yml

```diff
@@ -174,3 +176,26 @@ jobs:
         if: runner.os != 'Windows'
       - name: Run DOM to Database e2e tests
         run: npm run e2e:d2d -- --shard=${{ matrix.shardIndex }}/${{ matrix.shardTotal }}
+
+  deploy:
+    name: Deploy to abacus-test.nl
+    needs:
+      - backend
+    environment:
+      name: test
+      url: https://${{ github.sha }}.abacus-test.nl
+    permissions:
+      contents: read
+      deployments: write
```

Comment:
Why does this need deployment write access?

Reply:
It creates a new deployment ->  see the block below this comment "This branch was successfully deployed
2 active deployments"